apps/cowswap-frontend/src/modules/application/containers/App/index.tsx

```diff
@@ -51,10 +54,26 @@ const LinkComponent = ({ href, children }: PropsWithChildren<{ href: string }>)
 }
 
 export function App() {
-  useAnalyticsReporterCowSwap()
+  const { chainId, account } = useWalletInfo()
+  const { walletName } = useWalletDetails()
+  const cowAnalytics = useCowAnalytics()
+  const pixel = initPixelAnalytics()
```

Comment:
`pixel` is a static value and must be outside of the component function, otherwise it will cause unwanted rerenders.

Reply:
Addressed